modules/slugs/includes/qtranslate-slug-settings.php

```diff
@@ -102,8 +102,8 @@ function qts_show_form_field( $args = array() ) {
         case 'select':
             echo "<select id='$id' class='select$field_class' name='" . QTS_OPTIONS_NAME . "[$id]'>";
             foreach ( $choices as $item ) {
-                $value = esc_attr( $item, 'qts' );
-                $item  = esc_html( $item, 'qts' );
```

Comment:
It seems the old calls were wrong with the domain being sent as 2d argument, but these functions only take 1 arg.